spec/core/account/schemas/AccountDTO.yml

```diff
@@ -16,9 +16,17 @@ properties:
   addressHeight:
     $ref: "../../../schemas/Height.yml"
   publicKey:
-    $ref: "../../../schemas/PublicKey.yml"
+    allOf:
+      - description: >
+          The account's public key.
+          This information is only available once `publicKeyHeight` is greater than 0.
+      - $ref: "../../../schemas/PublicKey.yml"
   publicKeyHeight:
-    $ref: "../../../schemas/Height.yml"
+    allOf:
```

Comment:
It's an ugly workaround suggested by the OpenAPI Initiative guys themselves.
`$ref` nodes cannot have descriptions, and we cannot add the description to `Height.yml` because a `publicKeyHeight` is not the same thing as a `addressHeight`, even though both are Heights.
This is solved in the next version of OpenAPI, but it will take a while for all tools to support it. Meanwhile, the workaround is to add an `allOf`, so the `$ref` is merged with the `description`.
I tested the generated documentation and it looks as intended.

I also created an issue so we add these docs in all the missing places: #233

Reply:
If it doesn't break the client generators all good

https://github.com/nemtech/symbol-openapi-generator

Reply:
You're right, this breaks the generators. I've spent a few hours looking into it but this seems like a long-standing issue in `openapi-generator`.
Let's park this PR and hope that OpenAPI 3.1 is better supported...